engines/darkseed/configure.engine

```diff
@@ -0,0 +1,3 @@
+# This file is included from the main "configure" script
+# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
+add_engine darkseed "Darkseed" no "" "" ""
```

Comment:
```suggestion
add_engine darkseed "Darkseed" no "" "" "highres"
```

Reply:
Done. Just out of interest, what does this setting do?

Reply:
It automatically disables the engine on platforms that don't support resolutions >320x240. Since this engine uses a resolution of 640x350, disabling the engine reduces the size of the overall package (and the detection plugin) on platforms that don't support it.

Reply:
Nice, that makes sense